### ISSUE TITLE
React Storybook

### ISSUE THREAD

**Issue by HriBB** [tleunen/react-mdl#365]
https://github.com/kadirahq/react-storybook

Just an idea. Could be good for creating examples.

**Comment by tleunen** [comment]
Hi @HriBB, yes, I also believe it could be great, especially for the demos/examples. But I wonder how easy it is to customize the generated website to make it look the way you want.

**Comment by HriBB** [comment]
Not sure. It's just an idea. I've been playing around with it and it's really cool because you can easily create different scenarios of the same component.

I see they have [storybook-ui](https://github.com/kadirahq/storybook-ui). Maybe it's possible to create a custom UI like `react-mdl-storybook-ui`

Will do some research when I find some time.

**Comment by tleunen** [comment]
>  I've been playing around with it and it's really cool because you can easily create different scenarios of the same component.

I agree, I've also been playing with it for another UI library and it works very well. I'm really interested in using something like this (or like [carte-blanche](https://github.com/carteb/carte-blanche)) for react-mdl if we can customize the generated website.
